.gitignore

```diff
@@ -1,5 +1,6 @@
+.idea
```

Comment:
Remove this line. Ignoring `.idea` should be done in a global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
Thanks, done